odoo/tools/js_transpiler.py

```diff
@@ -537,10 +537,10 @@ def relative_path_to_module_path(url, path_rel):
 
 
 ODOO_MODULE_RE = re.compile(r"""
-    \s*                                       # some starting space  
+    \s*                                       # some starting space
     \/(\*|\/).*\s*                            # // or /*
     @odoo-module                              # @odoo-module
-    (\s+alias=(?P<alias>\S+))?                # alias=web.AbstractAction (optional)
+    (\s+alias=(?P<alias>[a-zA-Z\._\-]+))?     # alias=web.AbstractAction (optional)
```

Comment:
why not allow digits, e.g. `l10n_be`? Maybe `[\w.]+` does what you want in a simpler way?

play with it I guess: https://regex101.com/r/8h7Smp/1

Reply:
you're right